app/lib/surface/Label.coffee

```diff
@@ -46,17 +47,27 @@ module.exports = class Label extends CocoClass
     o = @buildLabelOptions()
     @layer.addChild @label = @buildLabel o
     @layer.addChild @background = @buildBackground o
+    @returnBounds = o.returnBounds
     @layer.updateLayerOrder()
 
   update: ->
     return unless @text and @sprite.sprite
     offset = @sprite.getOffset? (if @style in ['dialogue', 'say'] then 'mouth' else 'aboveHead')
-    offset ?= x: 0, y: 0  # temp (if not Lank)
+    offset ?= { x: 0, y: 0 }  # temp (if not Lank)
     offset.y += 10 if @style is 'variable'
     rotation = @sprite.getRotation()
     offset.x *= -1 if rotation >= 135 or rotation <= -135
     @label.x = @background.x = @sprite.sprite.x + offset.x
     @label.y = @background.y = @sprite.sprite.y + offset.y
+    if @returnBounds and @background.bitmapCache and camera = @sprite?.options?.camera
+      cache = @background.bitmapCache
+      width = (@background.bitmapCache.width - 20) * @background.scaleX
```

Comment:
Where does the `20` come from here, in general? Do you think it would it be overkill to turn it into a variable with an explanatory name to indicate what effect it has? 

EDIT: I see there's lots of numbers below, so might indeed be overkill.

Reply:
Yeah, imho it should be a constant. Again would be nice to rework it at all, because there are more hard-coded constants.